### ISSUE TITLE
GUI Wallet is not connected to daemon

### ISSUE THREAD

**Issue by joelstahre** [monero-project/monero-gui#1303]
I updated to Lithium Luna, im on Linux Ubuntu.
I Followed the upgrade instructions on getmonero.

I have the GUI version, so i just downloaded the new version to a new directory, and started it by doing 
 ```
./start-gui.sh
```

It loads my wallet file. (i can see my balance)

monerod is running

``` 
ps aux | grep mone
3199  8.4  3.3 1033524 130720 pts/20 Sl+  22:13   0:05 ./monero-wallet-gui
3494  0.0  0.0  15756   864 pts/19   S+   22:14   0:00 grep --color=auto mone
32448  1.6  4.3 82165380 170772 ?     Sl   18:08   4:10 /home/skynet/monero-gui-v0.12.0.0/monerod --detach --check-updates disabled

``` 

But the gui says that it is not connected.

```
GUI Wallet is not connected to daemon (Start deamon)
```

```
Network Status
Disconnected
```

Manage deamon (Status) output

```
Height: 1547643/1547643 (100.0%) on mainnet, not mining, net hash 420.55 MH/s, v7, up to date, 8(out)+4(in) connections, uptime 0d 4h 8m 7s

Height: 1547643/1547643 (100.0%) on mainnet, not mining, net hash 420.55 MH/s, v7, up to date, 8(out)+4(in) connections, uptime 0d 4h 8m 15s

Height: 1547651/1547651 (100.0%) on mainnet, not mining, net hash 425.19 MH/s, v7, up to date, 8(out)+4(in) connections, uptime 0d 4h 18m 8s

Height: 1547652/1547652 (100.0%) on mainnet, not mining, net hash 425.01 MH/s, v7, up to date, 8(out)+4(in) connections, uptime 0d 4h 18m 12s
```

**Comment by skftn** [comment]
For debugging purposes; which Ubuntu version are you on?

**Comment by joelstahre** [comment]
@skftn  im running Ubuntu 16.04 LTS

```
sudo netstat -tulpn | grep -i monero
tcp        0      0 0.0.0.0:18080           0.0.0.0:*               LISTEN      32448/monerod   
tcp        0      0 127.0.0.1:18081         0.0.0.0:*               LISTEN      32448/monerod   
tcp        0      0 127.0.0.1:18082         0.0.0.0:*               LISTEN      32448/monerod 
```

**Comment by joelstahre** [comment]
@dEBRUYNE-1 Yes, very similar. I also get ERROR logs saying

```
2018-04-08 10:43:36.934	    7f5af5d76840	ERROR	default	src/wallet/api/utils.cpp:46	error: std::bad_cast
```

**Comment by robione-nr** [comment]
Same issues. Same result from netstat

**Comment by robione-nr** [comment]
I'm above the average user but don't really have time to delve into things when they don't seem to go right. (Waaaay too many projects.) I installed `libunwind8` as that was my first error message running the wallet from that build, then `libqt5quick5` which was my second. I then had to download and install (it seemed) QT 5.7 and I did. It still didn't work.

Figured I'd just erase my wallets and start/recover  0.11.0.0 wallet with 0.12.0.0 daemon, setting ring size to 7 until it gets figured out..

**Comment by joelstahre** [comment]
@skftn any directions on what steps i need to do to test that?

**Comment by PanderMusubi** [comment]
https://github.com/monero-project/monero-gui/issues/1261 has a fix

**Comment by dEBRUYNE-1** [comment]
This particular issue is resolved in GUI v0.12.2.0: 

https://www.reddit.com/r/Monero/comments/8vkx2g/gui_v01220_released/

**Comment by dEBRUYNE-1** [comment]
+resolved
